client/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "private": true,
+  "name": "parent",
+  "version": "0.9.0",
+  "engines": {
+    "yarn": "1.0.x || >=1.2.1",
+    "node": ">=10.11.0 <13"
+  },
+  "devDependencies": {
+    "copyfiles": "latest",
+    "lerna": "^2.11.0",
+    "rimraf": "^2.6.1",
+    "tslint": "^5.5.0",
+    "typescript": "^3.9.2"
+  },
+  "scripts": {
+    "prepare": "lerna run prepare",
+    "watch": "lerna run --parallel watch",
+    "clean": "rimraf lib app/bundle.js app/bundle.js.map app/css",
+    "build": "tsc && yarn copyClient",
+    "copyClient": "copyfiles -f app/* ../org.eclipse.glsp.ide.workflow.editor/diagram"
```

Comment:
Please add a script for updating snapshot dependencies ("next) of sprotty @eclipse-glsp and @eclispe-glsp-examples.

See https://github.com/eclipse-glsp/glsp-client/blob/3d174f20df628ce7d0dcab452ce4771dbdb21467/package.json#L16 for reference